src/components/NavFooter.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { FaGithub } from 'react-icons/fa'; // You can use react-icons for the GitHub icon
+
+const Footer = () => {
+  return (
+    <footer style={footerStyle}>
+      <div style={leftStyle}>Color Conjure</div>
+      <div style={middleStyle}>
+        <p>
+          <a href="/disclaimer" style={linkStyle}>Disclaimer</a>: This website uses Seline for tracking, but it is privacy friendly and does not use cookies. More details can be found on their website
```

Comment:
We don't need to route to `/disclaimer` page, just the text on the same page should be good.